Product/Production/Deploy/pom.xml

```diff
@@ -479,7 +487,50 @@
         </plugins>
       </build>
     </profile>
-
+    <profile>
+	     <id>SBScripts</id>
+        <activation>
+          <activeByDefault>false</activeByDefault>
+        </activation>
+        <build>
+          <plugins>
+            <plugin>
+              <groupId>org.codehaus.mojo</groupId>
+              <artifactId>sql-maven-plugin</artifactId>
+              <version>1.5</version>
+            <dependencies>
+              <dependency>
+                <groupId>mysql</groupId>
+                <artifactId>mysql-connector-java</artifactId>
+                <version>5.1.10</version>
+              </dependency>
+            </dependencies>
+            <configuration>
+              <driver>com.mysql.jdbc.Driver</driver>
+              <url>jdbc:mysql://${db.host}</url>
+              <username>root</username>
+              <password>${mysql.root.password}</password>
+            </configuration>
+            <executions>
+              <execution>
+                <phase>process-test-resources</phase>
+                <goals>
+                  <goal>execute</goal>
+                </goals>
+                <configuration>
+                  <autocommit>true</autocommit>
+                  <srcFiles>
+                    <srcFile>../../../DBScripts/nhincdb/dropall.sql</srcFile>
```

Comment:
Also define path to dbscripts in CONNECT pom.xml

Reply:
Cannot establish an absolute path. Will go ahead with the present path assignment